src/lib/cli_lib/arg_type.ml

```diff
@@ -119,21 +119,18 @@ let user_command =
           Error.tag err ~tag:"Couldn't decode transaction id" |> Error.raise )
 
 module Work_selection_method = struct
-  [%%versioned
-  module Stable = struct
-    module V1 = struct
-      type t = Sequence | Random
-
-      let to_latest = Fn.id
-    end
-  end]
+  type t = Sequence | Random | Offset
+
+  let to_latest = Fn.id
 end
 
 let work_selection_method_val = function
   | "seq" ->
       Work_selection_method.Sequence
   | "rand" ->
       Random
+  | "roffset" ->
+      Offset
```

Comment:
*very minor nit*

up to you to change the type name to `Roffset`, but def not blocking fr me

Reply:
I changed for `Random_offset`, and also did other renamings